tools/local-network-setup/generate-config-files.js

```diff
@@ -45,6 +45,7 @@ for (let i = 0; i < numberOfNodes; i += 1) {
 
     parsedTemplate.rpcPort = 8900 + i;
     parsedTemplate.network.port = 9000 + i;
+    parsedTemplate.graphDatabase.name = `node${i}`;
```

Comment:
@kotlarmilos this is in order to have local network where all the nodes have unique graph db

Reply:
@nesovic Just one note, this won't work with Blazegraph, because it is not supporting different repository names, or at least we have not implemented that. So when using Blazegraph all nodes will use same DB

Reply:
It can stay, just to have that in mind when testing in local with blazegraph, that all nodes will use the same triple store

Reply:
It can stay, just to have that in mind when testing in local with blazegraph, that all nodes will use the same triple store